gtsfm/evaluation/compare_metrics.py

```diff
@@ -0,0 +1,81 @@
+"""Script to log metrics for comparison across SfM pipelines (GTSfM, COLMAP, etc.) as a JSON.
+
+Authors: Jon Womack
+"""
+import os
+import json
+from typing import Dict
```

Comment:
I dont think that is the right order, @johnwlambert ?

Reply:
Fixed.